client/reducers/login.js

```diff
@@ -0,0 +1,20 @@
+import { LOG_IN } from '../actions/types';
+
+const initialState = {
+    isLoggedIn: false
+};
+
+export default (state = initialState, action) => {
+    switch (action.type) {
+        case LOG_IN: {
+            const { username, name } = action.payload;
+            return {
+                isLoggedIn: true,
+                username,
+                name
```

Comment:
Do you expect `payload` to have other irrelevant properties? If not, then you can do this:

```js
return {
  isLoggedIn: true,
  ...payload
};
```

This also guarantees that any new data that you add to the response from your API is automatically included